scripts/transfer_files.sh

```diff
@@ -3,54 +3,56 @@
 usage() {
   echo -n "
 #############################################################
-# Upload files to Google Cloud Storage bucket.
+# Transer files to/from a Google Cloud Storage bucket.
 # 
 # This script will call the underlying gsutil utility, using
 # the parallel options (-m) by default. This will also write
-# a manifest log into the directory which can be used for upload
+# a manifest log into the directory which can be used for transfer
 # validation.
 #
 # For more information, see documentation at:
 # https://cloud.google.com/storage/docs/gsutil/commands/cp
 #
 # Note:
-# The SEARCH_PATH and DESTINATION are often similar. Normal invocation
+# The FROM_PATH and TO_PATH are often similar. Normal invocation
 # is done via $POCS/scripts/upload_image_dir.py. See also the utils in
 # $POCS/pocs/utils/google/storage.py.
 #
 # Note:
-# The SEARCH_PATH can be any pattern, e.g:
+# The transfer path can be any pattern, e.g:
 #     - 'PAN006'  # All images for unit
 #     - '*/Hd189733'  # All images for field
 #     - 'PAN006/Hd189733/7bab97' # All images for unit, field, camera
 #     - 'PAN006/*/7bab97' # All images for unit, camera
 #     - 'PAN006/Hd189733/7bab97/20180327T071126/' # Specific observation
 #############################################################
- $ $(basename $0) SEARCH_PATH DESTINATION MANIFEST_PATH
+ $ $(basename $0) FROM_PATH TO_PATH MANIFEST_FILE
  
  Options:
-  SEARCH_PATH     The string path to be searched.
-  DESTINATION     Remote upload destination.
-  MANIFEST_PATH   Path to manifest file for tracking uploads.
+  FROM_PATH     File(s) to be moved.
+  TO_PATH       Where file(s) are to be placed.
+  MANIFEST_FILE Path to manifest file for tracking uploads. Optional, defaults
```

Comment:
If a manifest file already exists, will it be overwritten?

Reply:
I've been trying to determine if it makes more sense to use the `rsync` function instead of the `cp` but can't seem to find any indication of which would be preferred. With the `rsync` we wouldn't have the manifest file, which might be nice. Currently they are accidentally included in the upload so they also exist in the bucket (unless that is how it is supposed to work).

Thoughts?